feature/experimental/p4rt/otg_tests/traceroute_packetin_test/README.md

```diff
@@ -1,34 +1,124 @@
 # P4RT-5.1: Traceroute: PacketIn
 
-
 ## Summary
 
-Verify that Traceroute packets are punted with correct metadata.
+Test FRR behaviors with encapsulation scenarios.
+
+## Topology
+
+ATE port-1 <------> port-1 DUT
+DUT port-2 <------> port-2 ATE
+DUT port-3 <------> port-3 ATE
+DUT port-4 <------> port-4 ATE
+DUT port-5 <------> port-5 ATE
+DUT port-6 <------> port-6 ATE
+DUT port-7 <------> port-7 ATE
+DUT port-8 <------> port-8 ATE
+
+## Baseline setup
+
+*   Setup equivalent to [TE-17.1 vrf_policy_driven_te](https://github.com/openconfig/featureprofiles/blob/main/feature/experimental/gribi/otg_tests/vrf_policy_driven_te/README.md), including GRibi programming.
+
+*   Install a BGP route resolved by ISIS in default VRF to route traffic out of DUT port-8 for 1.2.3.4.
```

Comment:
Can you pick a destination IP out of the expected ranges? https://github.com/openconfig/featureprofiles/blob/main/CONTRIBUTING.md#ip-addresses-assignment

Reply:
Done.